sktalk/corpus/write/writer.py

```diff
@@ -13,13 +15,50 @@ def write_json(self, path: str = "./file.json"):
         Write an object to a JSON file.
 
         Args:
-            name (str): The file name
-            directory (str): The path to the directory where the .json
-            file will be saved.
+            path (str): The path to the output file.
         """
         _path = Path(path).with_suffix(".json")
 
         object_dict = self.asdict()
 
         with open(_path, "w", encoding='utf-8') as file:
             json.dump(object_dict, file, indent=4)
+        print("Object saved to", _path)
+
+    def write_csv(self, path: str = "./file.csv"):
```

Comment:
Very minor detail, but I find it slightly confusing that `file.csv` is the default, whereas that literal filename will never be used.
Given that two files are required, I would perhaps split the argument into two for making their respective purposes explicit: directory and file prefix (and optionally suffix too). With reasonably defaults, this should not add more load to the caller, e.g.:

```suggestion
    def write_csv(self, directory: str = ".", prefix: str = "file", suffix: str = ".csv"):
```

In the implementation, you can now construct the different files like this:

```python
(Path(directory) / "_".join(prefix, specifier)).with_suffix(suffix)
```
with `specifier` being `"metadata"` or `"utterances"`.

Reply:
Thanks for this comment! I like to keep the workflow for csv saving and json saving similar, so I'm keeping the argument structure, but I've opted to use the given filename for the utterance db, and add a `_metdata` suffix only for the metadata csv file.